py_v_sdk/contract/atomic_swap_ctrt.py

```diff
@@ -2,9 +2,11 @@
 atomic_swap_ctrt contains Atomic Swap contract.
 """
 from __future__ import annotations
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Dict, Any, Union
```

Comment:
Usually importing a class directly is a bad practice while `typing` is an exception for readability reasons. Just FYI.